packages/amplify-provider-awscloudformation/src/display-helpful-urls.js

```diff
@@ -278,7 +280,24 @@ function showGraphQLTransformerMigrationMessage() {
   printer.info(`To get started, run 'amplify migrate api'`);
 }
 
+async function showGraphQLTransformerVersion(context) {
+  const meta = stateManager.getMeta();
+  const apiObject = (meta && meta.api) || {};
+  const hasGraphqlApi = !!Object.entries(apiObject)
+    .filter(([_, apiResource]) => apiResource.service === 'AppSync')
+    .map(([name]) => name).length;
+
+  if (!hasGraphqlApi) {
+    return;
+  }
+
+  const transformerVersion = await getTransformerVersion(context);
+
+  context.print.info(chalk`GraphQL transformer version: ${transformerVersion}`);
```

Comment:
nit: use `printer` from amplify-prompts

`chalk` without `()` looks like a typo possibly?

```js
chalk`GraphQL transformer version: ${transformerVersion}`
```

Reply:
> nit: use printer from amplify-prompts

I originally did, but changed it for easier mocking for the requested unit tests.

> chalk without () looks like a typo possibly?

That is the style used throughout that file.

Reply:
TIL that is valid JS.